### PR TITLE
added my info as GCI task

### PR DESCRIPTION
**Pull request by awesometushar000**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Replace this with your change description

### CODE REVIEW

**Comment by awesometushar000** [comment]
https://awesometushar000.github.io/gci17.fossasia.org/

**Review by the-codingninja — COMMENTED** [review]
In the preview link I cant see your image

**Comment by the-codingninja** [comment]
And please use image resizer to resize your image as your image is 480X800 it should be 240X240

**Review by akshatnitd — CHANGES_REQUESTED** [review]
@awesometushar000 Kindly resize the image size to 240x240 and send an updated demo link :)

**Comment by awesometushar000** [comment]
Please check what's wrong here?

**Review by abishekvashok — APPROVED** [review]

**Comment by abishekvashok** [comment]
@ms10398 i see that currently we have Jekyll down. Pls check it.

**Review by Achint08 — APPROVED** [review]

**Review by Achint08 — CHANGES_REQUESTED** [review]
Jekyll is down currently. Please check it.

**Review by the-codingninja — APPROVED** [review]

File: _data/students.yml [hunk 1]
```diff
@@ -90,17 +90,22 @@
...
   github: aryankedare
   image: aryankedare.jpg
   twitter: KedareAryan
-  
+
 - name: Samyak Jain
   github: samyakbvs
   image: samyak.png
   twitter: Samyakbvs
-  facebook: Samyak Jain
```
Comment (anshumanv) [inline _data/students.yml:103]: Please add this back, don't remove details of other students. Thanks.

Comment (awesometushar000) [reply _data/students.yml:103]: Done 👍

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
Thanks for being patient. :smile: 

As a best practice, please learn how to squash your commits and do so from next time on. :smile: 

Cheers!

**Comment by awesometushar000** [comment]
Thank You 
How did you recover Jekyll issue?
Please tell me

**Comment by the-codingninja** [comment]
You can have a look at this PR 
https://github.com/fossasia/gci17.fossasia.org/pull/138/files

**Pull request merged**